### DISCUSSION TITLE
after updating iris and many other mods with atlauncher the game will not open. blames fabric

### DISCUSSION THREAD

**Discussion by mrwubs42069** [FabricMC/fabric-api#3240] [Player support] [answered]
### Minecraft Logs

```txt
https://mclo.gs/JZRSK9L
```

### Description

ATLauncher kinda rapid fires when updating mods but I know iris needed an update. I started the updates through the launcher and it had many issues launching with multiple mods before I finally removed them and now it blames "'java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'fabric-networking-api-v1'!" ive tried reinstalling it through the launcher and manually. ive also tried older versions and the newest version and nothing has worked. I had to remove framework, BCLib, BetterEnd, BetterNether, Diagonal Fences, Easy Anvils, Easy Magic, Easy Shulker boxes, Puzzle, Puzzels Lib, and Very Many Players before finally getting to this point.

**Comment by Daomephsta** [comment, marked as answer]
Crash reports provide information, they do not assign blame.
`Could not execute entrypoint stage` is a very generic error. It occurs whenever anything goes wrong during setup of a particular mod. It does not mean that mod is at fault, it's just extra context for the whole error chain.

This is particularly important to remember when Mixin errors are involved. Java loads classes (individual code files inside mods and Minecraft) as needed, and Mixins are applied when the class they modify is loaded.
This means an invalid Mixin does not load when the mod that provides it does. It loads whenever its target class is first needed. This could be at literally any time. With mixin errors, any error in the chain above the first error mentioning Mixin, probably isn't relevant.

In an error chain, the bottom-most error is the original error. The errors above it are other code detecting that error and providing more context. So I recommend reading error chains bottom-up.

Reading bottom-up, there's a lot here, but all we're looking for is anything that looks like a mod ID. Carpet is mentioned a lot near the bottom, and you have version 1.4.113. Looking at Carpet's page, that version is for 23w31a. Try Carpet 1.4.112, which does support 1.20.1.

**Reply by mrwubs42069** [reply]
thx for the reply it worked i can load in again and adding the mods i removed back also works but now my world wont load just a blank skybox

**Reply by Poopooracoocoo** [reply]
@mrwubs42069 Farsight is not yet compatible with Sodium 0.5. You may want to remove it. See https://github.com/someaddons/farsight/issues/64

**Reply by mrwubs42069** [reply]
Thank you :) I made a new instance in atlauncher without that mod and some others and was able to copy everything over and get it working again I got my world back finally

**Reply by Poopooracoocoo** [reply]
Farsight, Sodium, and Iris are client side mods anyway so they won't affect your world :)

**Reply by mrwubs42069** [reply]
Yes that’s true I meant to say that I also accidentally loaded into my world without enabling bclib and better nether so the world gen probably corrupted or just couldn’t load
